bot/exts/moderation/infraction/infractions.py

```diff
@@ -151,6 +153,11 @@ async def send(*args, **kwargs) -> None:
         ctx.send = send
         await infr_manage_cog.infraction_append(ctx, infraction, None, reason=f"[Clean log]({log_url})")
 
+    @command()
+    async def compban(self, ctx: Context, user: UnambiguousMemberOrUser) -> None:
+        """Same as cleanban, but specifically for four days and with the ban reason used for compromised accounts."""
+        await self.cleanban(ctx, user, duration=arrow.utcnow() + AUTO_BAN_DURATION, reason=AUTO_BAN_REASON)
```

Comment:
`arrow.utcnow() + AUTO_BAN_DURATION` results in an arrow object, which does not match the `Expiry` type hint.

This doesn't seem like an issue now, but may be broken in a future change, as an `arrow` object is not marked as supported.

Either we need to add `arrow` as part of the `Expiry` union, or change this to instead pass a datetime to cleanban.

Reply:
Oh, I copied that from this the following snippet. Perhaps I misunderstood what that does, I'll change that.

https://github.com/python-discord/bot/blob/e1967b58b859578b1702d873cbfa737ea3b2feaa/bot/exts/filters/filtering.py#L416

Reply:
Ah, it's wrong there too then 😅. Feel free to make the same change there too if you like.

Reply:
fixed in https://github.com/python-discord/bot/pull/2250/commits/a7234a824f3d4b2d052357d73d23a6e0c5cbb1b6